### PR TITLE
Banner inner

### PR DESCRIPTION
**Pull request by denizmrtoglu**

### CODE REVIEW

File: app/components/banner-left.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  top: -7.8rem;
+}
+
+.balls-img {
+  position: relative;
+  width: 31rem;
+  top: 1.2rem;
+}
+
+/* Mobil (640px ve altı) */
```
Comment (denizmrtoglu) [inline app/components/banner-left.css:13]: Translate Turkish comment lines into English or remove them.

File: app/components/banner-center.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <p className="bg-pink-400 text-white px-2 py-1 rounded-r-lg">
+              Fun as a real gift
+            </p>
+          </div>
+          <div className="flex flex-col items-center space-y-2">
+            <p className="text-gray-700">Collects money from more friends</p>
+            <p className="text-gray-700">Can be spent anywhere as cash</p>
+          </div>
+        </div>
+        <div className="w-full md:w-2/3 h-auto phone flex justify-center items-end md:justify-end md:items-center md:h-2/3">
```
Comment (denizmrtoglu) [inline app/components/banner-center.tsx:32]: We should avoid using both tailwind and css. Prefer one.

File: app/components/banner-left.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  }
+  .balls-img {
+    left: -4rem;
+    width: 20rem;
+    top: 0.6rem;
+  }
+}
+
+/* Tablet (641px ila 1024px arası) */
+@media (min-width: 641px) and (max-width: 1024px) {
```
Comment (denizmrtoglu) [inline app/components/banner-left.css:28]: I think we don't need to use media break, Tailwind can handle it.

File: app/components/banner-right.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.description-text{
+
```
Comment (denizmrtoglu) [inline app/components/banner-right.css:2]: Remove the empty line. You can use prettier for this.

**Comment by denizmrtoglu** [comment]
Why are all the files in the same place? Research and implement best practice.

**Review by TunahanBerber — COMMENTED** [review]
These changes were made to make the banner content look better. Please review and approve.

**Pull request merged**